ament_copyright/ament_copyright/main.py

```diff
@@ -218,7 +218,9 @@ def main(argv=sys.argv[1:]):
 
 def add_missing_header(file_descriptors, name, license_, verbose):
     copyright_ = 'Copyright %d %s' % (int(time.strftime('%Y')) - 1 + 1, name)
-    header = license_.file_header.format(**{'copyright': copyright_})
+    header = license_.file_header.format(**{
+        'copyright': copyright_,
+        'copyright_holder': 'the copyright holder'})
```

Comment:
"the copyright holder" doesn't seem to be a reasonable replacement. Either it should be an actual name or the template should use that literal string instead of a placeholder.

Reply:
Was thinking a little about it just now, and actually the solution is to use the 'name' variable passed as argument. This way the generated copyright might look as this one:
[class_loader.cpp](https://github.com/ros/class_loader/blob/melodic-devel/src/class_loader.cpp)

Where the copyright in the top has the copyright year autogenerated, and the passed name is used in both places